src/main.cpp

```diff
@@ -30,6 +30,16 @@ SOFTWARE.
 #include "AudioOutputI2S.h"
 #include <EasyButton.h>
 
+
+// ********* User settings *********
+const char* ssid = "yourssid";
+const char* password = "yourpassw0rd";
+const char* audioStreamUrl = "http://0n-80s.radionetz.de:8000/0n-80s.mp3";
+const uint8_t countdownSeconds = 120;
```

Comment:
What is the official recommendation for brushing the teeth? I thought it's 180 seconds?